lib/redcloth.rb

```diff
@@ -10,6 +10,11 @@
   conf = Object.const_get(defined?(RbConfig) ? :RbConfig : :Config)::CONFIG
   prefix = conf['arch'] =~ /mswin|mingw/ ? "#{conf['MAJOR']}.#{conf['MINOR']}/" : ''
   lib = "#{prefix}redcloth_scan"
+  begin 
```

Comment:
@mohits : there is another suggestion for this fix in #82 .

I am looking into the current PRs open and trying to merge them all.
Please let me know if you have a bit of time to review it for possible issues.

I commented in there as well that I find your code more safer, given I am still situating myself, and still moving slower.

Reply:
Hi @heliocola - I put this gem down in the list of things that I want to get back to and work on this year... just didn't expect it to fire up this early in the year. Let me try to put things back together and try to review the requirements again, and where I had stopped.

Reply:
Thank you @mohits !

Reply:
@mohits : FYI: I will merge this one as it helps @MSP-Greg in his windows system and this is very safe.
At your own time, please feel free to jump back in.

Reply:
Hi, I hope to be able to start looking at this in February again. I think I lost the VM on which I had the whole setup, and need to set it up to ensure that everything works correctly. For now, I think #51 and #61 can be closed with this. 

I'm happy to collaborate, at least in reviewing the issues and doing triage, and helping to close the irrelevant ones.

Reply:
Thank you @mohits .
I will close #61 and if needed you can open a new one, when you get to it.
Take your time!